.eslintrc.js

```diff
@@ -64,10 +65,31 @@ module.exports = {
         },
       },
       rules: {
+        'no-duplicate-imports': [0], // handled by @typescript-eslint
         'require-atomic-updates': [0],
-        'import/no-unresolved': [0],
+        'import-helpers/order-imports': [
+          'error',
+          {
+            newlinesBetween: 'always',
+            groups: [['module'], ['parent', 'sibling', 'index']],
+          },
+        ],
+        'import/no-unresolved': [
+          'error',
+          {
+            ignore: [
+              'netlify-cms-lib-util',
+              'netlify-cms-lib-widgets',
+              'netlify-cms-backend-github',
+              'netlify-cms-backend-gitlab',
+              'netlify-cms-backend-bitbucket',
+            ],
+          },
+        ],
+        '@typescript-eslint/ban-types': [0], // TODO enable in future
```

Comment:
This rule forbids using empty object as type. Probably good idea to enable it in future.